src/Storefront/Resources/views/storefront/layout/meta.html.twig

```diff
@@ -98,11 +98,11 @@
         {% endblock %}
 
         {% block layout_head_title %}
-            <title itemprop="name">
-                {% block layout_head_title_inner %}{% apply spaceless %}
+            <title itemprop="name">{% apply spaceless %}
+                {% block layout_head_title_inner %}
                     {{ metaTitle }}
-                {% endapply %}{% endblock %}
```

Comment:
Why not `{%- block name -%}{{ var }}{%- endblock -%}`?

Reply:
If people accidentally add spaces on the administration, this fixes it 🙃

Reply:
You're missing out the `-` from my example. It trims the spaces as well.

Reply:
Thanks! TIL! No clue, we can change that, not sure the twig version is new enough, will test

Reply:
`{%- ... -%}` is just a short cut for `{% apply spaceless %} ... {% endapply %}` 
and in this case i would go for clarity over brevity as apply spaceless is IMHO way easier to understand especially for newcomers who stumble over that code path

Reply:
According to the docs, [Whitespace Control](https://twig.symfony.com/doc/3.x/templates.html#whitespace-control) and [spaceless filter](https://twig.symfony.com/doc/3.x/filters/spaceless.html) aren't quite the same, as in trim vs regex to trim spaces between html tags as well.

Reply:
ah nice for the hint i didn't know that, but in this case the result should be the same